qcengine/programs/nwchem/harvester.py

```diff
@@ -614,11 +614,103 @@ def harvest_outfile_pass(outtext):
     if mobj:
         psivar["N ALPHA ELECTRONS"] = mobj.group(2)
         psivar["N BETA ELECTRONS"] = mobj.group(3)
+
+        if psivar["N ALPHA ELECTRONS"] == psivar["N BETA ELECTRONS"]:
+
+            # get HOMO and LUMO energy
+            mobj = re.search(
+                r"Vector"
+                + r"\s+"
+                + r"%d" % (psivar["N ALPHA ELECTRONS"])
+                + r"\s+"
+                + r"Occ="
+                + r".*"
+                + r"\s+"
+                + r"E="
+                + r"([+-]?\s?\d+[.]\d+)"
+                + r"[D]"
+                + r"([+-])"
+                + r"[0]"
+                + r"(\d+)",
+                outtext,
+                re.MULTILINE,
+            )
+            if mobj:
+                if mobj.group(2) == "+":
+                    lumo = float(mobj.group(1)) * (10 ** (-1 * float(mobj.group(3))))
+                    psivar["HOMO"] = np.array([round(lumo, 10)])
+                else:
+                    lumo = float(mobj.group(1)) * (10 ** (-1 * float(mobj.group(3))))
+                    psivar["HOMO"] = np.array([round(lumo, 10)])
```

Comment:
am I missing how the `else` block differs from the `if` block?

Reply:
Sorry, my student and I used this for testing.  I just removed the if-else blocks.

Reply:
Actually just spoke to my student.  At first, he told me it was just for testing but the reason for the IF else is to detect the sign of the power e.g "2.125011D-01",  I'm changing it again.  Sorry about that